wist-scoreboard-generator/src/app/app-routing.module.ts

```diff
@@ -1,7 +1,12 @@
+import { HomepageComponent } from './Pages/homepage/homepage.component';
 import { NgModule } from '@angular/core';
 import { RouterModule, Routes } from '@angular/router';
+import { GameSettingsPageComponent } from './Pages/game-settings-page/game-settings-page.component';
 
-const routes: Routes = [];
+const routes: Routes = [
+  {path:'', component: HomepageComponent},
+  {path:'game', component: GameSettingsPageComponent}
```

Comment:
I would redirect any non matching path to the home page. see: https://angular.io/guide/router#setting-up-redirects